build/phase.go

```diff
@@ -118,19 +120,18 @@ func (p *Phase) Run(context context.Context) error {
 		return errors.Wrapf(err, "failed to create '%s' container", p.name)
 	}
 	p.appOnce.Do(func() {
-		var mode int64 = -1
-		if runtime.GOOS == "windows" {
-			mode = 0777
-		}
+		var appReader io.ReadCloser
+		appReader, err = p.createAppReader()
+		defer appReader.Close()
 
-		appReader, errChan := archive.CreateTarReader(p.appDir, appDir, p.uid, p.gid, mode)
-		if err = p.docker.CopyToContainer(context, p.ctr.ID, "/", appReader, types.CopyToContainerOptions{}); err != nil {
-			err = errors.Wrapf(err, "failed to copy files to '%s' container", p.name)
+		if err != nil {
+			err = errors.Wrapf(err, "create tar archive from '%s'", p.appPath)
 		}
 
-		err = <-errChan
 		if err != nil {
```

Comment:
I think you want the opposite conditional here

Reply:
Haha, yeah... We saw that too